components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -362,7 +365,7 @@ export const DrawerNavGroup: React.FC<DrawerNavGroupProps> = (props) => {
     return (
         <>
             <List
-                style={{ backgroundColor }}
+                style={{ backgroundColor, paddingBottom: 0 }}
```

Comment:
padding top 0 too?

Reply:
I think only paddingBottom had the padding issue.

Reply:
ok just wondering...most places we use List we do padding: 0 (not sure why MUI puts padding on their Lists by default)